crates/runtime/src/runtime.rs

```diff
@@ -54,6 +54,26 @@ impl RuntimeBuilder {
             force_pooling_allocator: false,
         }
     }
+    /// Returns a custom Max Execution Time [`RuntimeBuilder`]
+    pub fn custom(max_exec_time: Duration) -> Self {
```

Comment:
how about we add a `max_execution_time` method on `RuntimeBuilder` instead?

Reply:
Actually, I see we already have it, why would we need the `custom` here?

Reply:
> Actually, I see we already have it, why would we need the `custom` here?

That makes so much more sense, yep we don't need a custom RunBuilder generator _yet_ (till there's a need to make things more granular, which is still farfetched)